src/modules/markets/actions/load-markets-info-if-not-loaded.js

```diff
@@ -0,0 +1,11 @@
+import { loadMarketsInfo } from 'modules/markets/actions/load-markets-info'
+import { isMarketLoaded } from 'modules/market/helpers/is-market-loaded'
+import logError from 'utils/log-error'
+
+export const loadMarketsInfoIfNotLoaded = (marketIds, callback = logError) => (dispatch, getState) => {
+  const { marketsData } = getState()
+  const marketIdsToLoad = marketIds.filter(marketId => isMarketLoaded(marketId, marketsData))
+  console.log('marketIdsToLoad:', marketIdsToLoad)
```

Comment:
const marketIdsToLoad = marketIds.filter(marketId => isMarketLoaded(marketId, marketsData))

should it be `!isMarketLoaded` not market loaded? I made this change and now getting market data loaded. wanted to check with you.

Reply:
Good catch, yes it is backwards here!